src/validation/presenter.validation.ts

```diff
@@ -2,7 +2,9 @@ import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
 
+const VALID_CHARACTERS = /^[-,.:; 0-9A-Z&@$£¥€'"«»''""?!/\\()[\]{}<>*=#%+ÀÁÂÃÄÅĀĂĄÆǼÇĆĈĊČÞĎÐÈÉÊËĒĔĖĘĚĜĞĠĢĤĦÌÍÎÏĨĪĬĮİĴĶĹĻĽĿŁÑŃŅŇŊÒÓÔÕÖØŌŎŐǾŒŔŖŘŚŜŞŠŢŤŦÙÚÛÜŨŪŬŮŰŲŴẀẂẄỲÝŶŸŹŻŽa-zÀÖØſƒǺǿẀẅỲỳ]*$/;
```

Comment:
Having this here is probably fine for this PR as it's the first occurrence of its use, but in the next PR, we don't want to keep duplicating this string so we'll have to then move it somewhere central and work out how to get validation working with the imported constant value.